tests/aws/services/stepfunctions/v2/test_sfn_api_validation.py

```diff
@@ -0,0 +1,70 @@
+import json
+
+import pytest
+
+from localstack.aws.api.stepfunctions import StateMachineType
+from localstack.testing.pytest import markers
+from tests.aws.services.stepfunctions.templates.callbacks.callback_templates import (
+    CallbackTemplates,
+)
+from tests.aws.services.stepfunctions.templates.validation.validation_templates import (
+    ValidationTemplate,
+)
+
+
+@markers.snapshot.skip_snapshot_verify(
+    paths=[
+        "$..tracingConfiguration",
```

Comment:
Any particular reason for this here? I didn't see that key anywhere in the recorded snapshots below